scipy/special/_special_ufuncs_docs.cpp

```diff
@@ -4762,6 +4762,42 @@ const char *mathieu_cem_doc = R"(
     --------
     mathieu_a, mathieu_b, mathieu_sem
 
+    Notes
+    -----
+    Even Mathieu functions are the solutions to Mathieu's differential equation
+
+    .. math::
+
+        \frac{d^2y}{dx^2} + (a_m - 2q \cos(2x))y = 0
+
+    for which the characteristic number :math:`a_m` results in an even, periodic
+    solution :math:`y(x)`` with period 180 degrees (for even :math:`m`) or
+    360 degrees (for odd :math:`m`).
+
+    References
+    ----------
+    'Mathieu function'. *Wikipedia*.
+    https://en.wikipedia.org/wiki/Mathieu_function
+
+    Examples
+    --------
+    Plot Mathieu functions of orders ``2`` and ``4``.
+
+    >>> import numpy as np
+    >>> from scipy import special
+    >>> import matplotlib.pyplot as plt
+    >>> m = np.asarray([2, 4])
+    >>> q = 50
+    >>> x = np.linspace(-180, 180, 300)[:, np.newaxis]
+    >>> y, _ = special.mathieu_cem(m, q, x)
+    >>> plt.plot(x, y)
+    >>> plt.xlabel('x (degrees)')
+    >>> plt.ylabel('y')
+    >>> plt.legend(('m = 2', 'm = 4'))
+
+    Note that the functions are even. Also, because the orders ``2`` and
```

Comment:
```suggestion
    Because the orders ``2`` and
```

Reply:
Wait, why remove this? This is an important characteristic of the function. 
https://en.wikipedia.org/wiki/Even_and_odd_functions#:~:text=Even%20functions%20are%20those%20real,with%20respect%20to%20the%20origin.

Reply:
Oh, I guess it would be in the first sentence then. Ok.